### PR TITLE
fixes for tools repo merge

### PR DESCRIPTION
**Pull request by zramsay**
continues #1943 but still running into `dep` problems

* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG.md

### CODE REVIEW

File: rpc/lib/doc.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ Each route is available as a GET request, as a JSONRPCv2 POST request, and via J
 
 # Examples
 
-* [Tendermint](https://github.com/tendermint/tendermint/blob/master/rpc/core/routes.go)
-* [tm-monitor](https://github.com/tendermint/tools/blob/master/tm-monitor/rpc.go)
+* [Tendermint](https://github.com/tendermint/tendermint/blob/develop/rpc/core/routes.go)
```
Comment (melekes) [inline rpc/lib/doc.go]: why change to `develop`?

File: tools/tm-monitor/Gopkg.toml [hunk 2]
```diff
@@ -37,21 +37,13 @@
...
   version = "~0.9.6"
 
 [[constraint]]
   name = "github.com/tendermint/tendermint"
-  version = "0.19.0"
-
-[[override]]
-  name = "github.com/tendermint/tmlibs"
-  version = "~0.8.2-rc1"
+  version = "develop"
```
Comment (melekes) [inline tools/tm-monitor/Gopkg.toml]: branch = "master"

File: rpc/lib/doc.go (hunk 1 above)
Comment (zramsay) [reply rpc/lib/doc.go]: for consistency

Comment (melekes) [reply rpc/lib/doc.go]: doc should point to the latest stable release - master

Comment (zramsay) [reply rpc/lib/doc.go]: fixed

File: tools/tm-monitor/Gopkg.toml [hunk 3]
```diff
@@ -37,21 +37,13 @@
...
   version = "~0.9.6"
 
 [[constraint]]
   name = "github.com/tendermint/tendermint"
-  version = "0.19.0"
-
-[[override]]
-  name = "github.com/tendermint/tmlibs"
-  version = "~0.8.2-rc1"
+  version = "master"
```
Comment (melekes) [inline tools/tm-monitor/Gopkg.toml]: `branch = "master"`

File: tools/tm-monitor/rpc.go [hunk 4]
```diff
@@ -16,7 +16,7 @@ func startRPC(listenAddr string, m *monitor.Monitor, logger log.Logger) {
 	wm := rpc.NewWebsocketManager(routes, nil)
 	mux.HandleFunc("/websocket", wm.WebsocketHandler)
 	rpc.RegisterRPCFuncs(mux, routes, cdc, logger)
-	if _, err := rpc.StartHTTPServer(listenAddr, mux, logger); err != nil {
+	if _, err := rpc.StartHTTPServer(listenAddr, mux, logger, rpc.Config()); err != nil {
```
Comment (melekes) [inline tools/tm-monitor/rpc.go]: `rpc.Config{}`, no?

**Comment by ValarDragon** [comment]
I think this is now good to merge

**Pull request merged**